src/app/views/query-runner/QueryRunner.tsx

```diff
@@ -5,7 +5,7 @@ import { connect } from 'react-redux';
 import { bindActionCreators, Dispatch } from 'redux';
 
 import { telemetry } from '../../../telemetry';
-import { RUN_QUERY_EVENT } from '../../../telemetry/event-types';
+import { RUN_QUERY_EVENT, DROPDOWN_CHANGE_EVENT } from '../../../telemetry/event-types';
```

Comment:
does not look like this is used in this file.

Reply:
It's being used to track version change.